src/gc/gcconfig.h

```diff
@@ -95,6 +95,8 @@ class GCConfigStringHolder
       "Specifies the ratio compacting GCs vs sweeping")                                        \
   INT_CONFIG(GCHeapAffinitizeMask, "GCHeapAffinitizeMask", 0,                                  \
       "Specifies processor mask for Server GC threads")                                        \
+  STRING_CONFIG(GCHeapAffinitizeRanges, "GCHeapAffinitizeRanges",                              \
+      "Specifies list of processors for Server GC threads")                                    \
```

Comment:
can you please indicate the format here since it's not obvious?

Reply:
Fixed